backend/tests/services/knowledge/code_wiki/test_code_wiki_ghost.py

```
@@ -101,11 +117,18 @@ def test_component_hierarchies_have_substantive_overview_pages(system_prompt: st
 def test_high_value_relationships_require_diagram_consideration(system_prompt: str):
+    normalized_prompt = " ".join(system_prompt.split())
+
     for subject in ("architecture", "cross-component", "lifecycles", "data models"):
         assert subject in system_prompt
-    assert "Use Mermaid diagrams" in system_prompt
+    assert "Use source-grounded Mermaid" in system_prompt
     assert "code-wiki-mermaid" not in system_prompt
-    assert "decorative diagrams" in system_prompt
+    assert "multi-boundary architecture" in system_prompt
+    assert "system-context map" in system_prompt
+    assert "start with 4–8 major nodes" in system_prompt
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)